dear_petition/petition/models.py

```diff
@@ -168,12 +168,15 @@ def refresh_record_from_data(self):
                 disposition_method=offenses.get("Disposition Method", ""),
             )
             offense_records = offenses.get("Records", [])
+            if not created:
```

Comment:
We should probably call this regardless of the value of `created`, because whether we're creating a new `Offense` here or not, we don't want other `OffenseRecord`s for this `Offense` hanging out in the database.

Reply:
done